Content.Server/Projectiles/ProjectileSystem.cs

```diff
@@ -37,7 +37,18 @@ private void OnStartCollide(EntityUid uid, ProjectileComponent component, ref St
 
         var target = args.OtherEntity;
         // it's here so this check is only done once before possible hit
-        var attemptEv = new ProjectileReflectAttemptEvent(uid, component, false);
+	// MOFFSTATION - Ricochet
```

Comment:
```suggestion
	// Moffstation - Begin - Ricochet
```

Just Some Style Guide Things